spec/models/manageiq/providers/kubevirt/inventory/parser_spec.rb

```diff
@@ -45,7 +45,18 @@
       parser.instance_variable_set(:@vm_os_collection, os_collection)
       parser.instance_variable_set(:@disk_collection, disk_collection)
 
-      parser.send(:process_template, unprocessed_object("template.json"))
+      # TODO: check whether this is the format that fog-kubevirt would return
```

Comment:
is this an important #TODO? if so maybe it should be done now, it looks like a simple puts can check it. Else maybe it is not important and then we can drop it.

Reply:
Old comment, will remove